apple-pie/index.html

```diff
@@ -5,17 +5,71 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
     <!-- don't forget to link your styles -->
+    <link rel="stylesheet" href="/lab-css-recipes-clone/apple-pie/styles/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans" rel="stylesheet">
   </head>
   <body>
-    Apple Pies This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
-    be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
-    favorites as well! Ingredients 1 recipe pastry for a 9 inch double crust pie 1/2 cup unsalted butter 3 tablespoons
-    all-purpose flour 1/4 cup water 3 tablespoons all-purpose flour 1/4 cup water 1/2 cup white sugar 1/2 cup packed
-    brown sugar 8 Granny Smith apples - peeled, cored and sliced Directions Preheat oven to 425 degrees F (220 degrees
-    C). Melt the butter in a saucepan. Stir in flour to form a paste. Add water, white sugar and brown sugar, and bring
-    to a boil. Reduce temperature and let simmer. Place the bottom crust in your pan. Fill with apples, mounded
-    slightly. Cover with a lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-    that it does not run off. Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
-    degrees C). Continue baking for 35 to 45 minutes, until apples are soft.
+
+    <section id="header">
+      <div class="title">
+        <h1>Apple Pie</h1>
+      </div>
+
+    </section>
+    
+    <section id="main-content">
+      <article>
+        <div>
+          <h4>This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your favorites as well!</h3>
+          
+          <div class="img-container">
+            <img class="info-img" src="/lab-css-recipes-clone/apple-pie/images/recipe-info.png"/>
+          </div>
+            
+          <hr>
+        </div>
+      </article>
+      
+      <article>
+        <div>
+          <h3>Ingredients</h3>
+
+          <ul>
+            <li>- 1 recipe pastry for a 9 inch double crust pie</li>
```

Comment:
if you do it this way with the minus, add a list-style: none; to the ul element.

Reply:
Yes, I searched for the code but since we didn't mentioned at the class I cheated a little bit☺.